File: ai/openai/nodejs/completion1/chat1.js
import OpenAI from "openai";

const openai = new OpenAI();

async function main() {
  const completion = await openai.chat.completions.create({
    messages: [{ role: "system", content: "Qual é a capital do Brasil?" }],
    model: "gpt-3.5-turbo",
  });

  console.log(completion.choices[0]);
}

main();

File: ai/openai/python/completion1/chat1.py
from openai import OpenAI
client = OpenAI()

completion = client.chat.completions.create(
  model="gpt-3.5-turbo",
  messages=[
    {"role": "system", "content": "Qual a melhor linguagem de programação do mundo?"},
    #{"role": "user", "content": ""}
  ]
)

print(completion.choices[0].message)

